### ISSUE TITLE
[DS]

### ISSUE THREAD

**Issue by Mallik813** [frextrite/Data-Structures-Algorithms-Hacktoberfest-2K19#206]
# Data Structure

- [ ] I have gone through the repository and made sure this data structure has not been implemented

## I have seen that in the BST code implemented through cpp, there's only **INSERT** function and **inorder** traversal. **I want to add more functions like PREORDER traversal, POST ORDER traversal and also delete nodes using C language.**

- **Data Structure: BINARY SEARCH TREE** 

- **Programming Language: C** 

**Is this your first contribution to this repository?**
Yes

**Comment by frextrite** [comment]
Hey @Mallik813 I think these traversals were just added. Check `Data Structures/Binary Tree Traversals/`

**Comment by Mallik813** [comment]
@frextrite Yes they are added. But in python langauage. Most of the competetive programmers use c++ as their language. And python codes generally have a lot of time complexity and different from c++ code.
So I think there will be not much problem even if that exists in two languages.

Thanks.

**Comment by frextrite** [comment]
Sure go ahead @Mallik813
